restapi/src/main/java/io/stargate/web/impl/Server.java

```diff
@@ -133,7 +135,20 @@ protected void configure() {
     environment.jersey().register(IndexesResource.class);
 
     // Documents API
+    JsonConverter jsonConverter = new JsonConverter(environment.getObjectMapper());
```

Comment:
another suggestion:

1. let's register all classes, `JsonConverter` as well
2. let's extract all of this into a DocsApi bBinder class

``` 
public class DocsApiComponentsBinder extends AbstractBinder {

    private final Environment environment; // if needed, I guess not

    protected void configure() {
        /// all services and resources needed for the docs api
        ....
    }

} 

and then in the server just:

```
environment.jersey().register(new DocsApiComponentsBinder(environment));
```

clean and easy